l10n_th_address/__init__.py

```diff
@@ -2,3 +2,15 @@
 # © 2017 Ecosoft (ecosoft.co.th).
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 from . import models
+from odoo import api, SUPERUSER_ID
+
+
+def _update_data_noupdate(cr, registry):
```

Comment:
@dreispt really not sure what you mean by won file. Beside, from what I see from other OCA addons, most if not all are doing in __init__.py, isn't it?